themes/theme-core/package.json

```diff
@@ -25,6 +25,7 @@
     "@marigold/components": "workspace:*"
   },
   "devDependencies": {
+    "@marigold/components": "workspace:*",
```

Comment:
Is it ok to have it as peer and dev dependency?

Reply:
Has to because peer steps are not installed. This isn't a change from the current structure